airbyte-integrations/connectors/source-hubspot/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 36c891d9-4bd9-43ac-bad2-10e12756272c
-  dockerImageTag: 4.8.0-rc.1
+  dockerImageTag: 4.9.0
```

Comment:
we need to update this to `5.0.0` to align with our version increment rules

Reply:
yep will fix